src/lib/esnext.typedarrays.d.ts

```diff
@@ -0,0 +1,69 @@
+interface Uint8Array<TArrayBuffer extends ArrayBufferLike> {
+    /**
+     * Converts the `Uint8Array` to a base64-encoded string.
+     * @param options If provided, sets the alphabet and padding behavior used.
+     * @returns A base64-encoded string.
+     */
+    toBase64(options?: {
+        alphabet?: "base64" | "base64url";
+        omitPadding?: boolean;

```

Comment:
The options types need to look like:


```suggestion
        alphabet?: "base64" | "base64url" | undefined;
        omitPadding?: boolean | undefined;
```

In order to handle `exactOptionalPropertyTypes`.